introduction/07-faq.asciidoc

```diff
@@ -0,0 +1,15 @@
+== FAQ
+
+To conclude the Introduction segment of the Learning Path, here are some Frequently Asked Questions people have when embarking on their InnerSource journey.
+
+=== What is the cost/overhead?
+
+=== Why not just open source?
+
+=== Will this slow us down?
```

Comment:
It depends on how far you're going.  You'll probably be going a lot further than you think.

![image](https://user-images.githubusercontent.com/9609562/151901209-52b3468b-dedd-4319-9ca3-38b6b2bcfaf5.png)

Reply:
https://github.com/InnerSourceCommons/InnerSourceLearningPath/pull/465/commits/0e4a7d0a88dfae20331af1a86b47e3b1ba80eef2